quick_tour/metadata_filtering.ipynb

```diff
@@ -0,0 +1,2904 @@
+{
```

Comment:
<p>Can you make it : </p><p>'Metadata filtering is a new feature in Pinecone that allows you to apply filters on vector search based on metadata'</p><p>and for the last line :</p><p>'In this notebook, we will walk through a simple use of filtering while performing vector search on documents'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>